.travis.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       compiler: clang
       env:
       before_install:
-        - sudo pkg install -y gtk-layer-shell gtkmm30 jsoncpp libdbusmenu
+        - sudo pkg install -y gtk-layer-shell gtkmm30 jsoncpp libdbusmenu sndio
```

Comment:
```suggestion
        - export CPPFLAGS=-isystem/usr/local/include LDFLAGS=-L/usr/local/lib # sndio
        - sudo pkg install -y gtk-layer-shell gtkmm30 jsoncpp libdbusmenu sndio
```

Reply:
Why not use `-I`?

Thanks for testing and for the link to CI results! Do you use `sndio`? Would be nice to get opinions on usability and such.

Reply:
> Why not use `-I`?

`-isystem` puts include path after `-I` (avoiding conflicts with local headers) and silences warnings (that cannot be fixed by this project).

See also https://gcc.gnu.org/onlinedocs/cpp/System-Headers.html

> Do you use `sndio`? Would be nice to get opinions on usability and such.

No. Maybe @t6 can help.

Reply:
I see, thanks!